load-testing/tests/relays_stress_test.go

```diff
@@ -511,20 +510,25 @@ func (s *relaysSuite) ALoadOfConcurrentRelayRequestsAreSentFromTheApplications()
 	<-s.ctx.Done()
 }
 
+func (s *relaysSuite) TheUserShouldWaitForSeconds(dur int64) {
+	time.Sleep(time.Duration(dur) * time.Second)
+}
+
 func (s *relaysSuite) TheCorrectPairsCountOfClaimAndProofMessagesShouldBeCommittedOnchain() {
+	logger.Info().
+		Int("claims", s.currentClaimCount).
+		Int("proofs", s.currentProofCount).
+		Msg("Claims and proofs count")
+
 	require.Equal(s,
 		s.currentClaimCount,
 		s.currentProofCount,
 		"claims and proofs count mismatch",
 	)
 
-	logger.Info().
-		Int("claims", s.currentClaimCount).
-		Int("proofs", s.currentProofCount).
-		Msg("Claims and proofs count")
 	// TODO_TECHDEBT: The current counting mechanism for the expected claims and proofs
 	// is not accurate. The expected claims and proofs count should be calculated based
-	// on the effectively sent relay requests.
+	// on a function of(time_per_block, num_blocks_per_session) -> num_claims_and_proofs.
```

Comment:
I'm struggling with this a bit. What does `time_per_block` have to do with it? I would've expected it to be a function of the sum of the number of application/supplier permutations in each session (integrating as actors are scaled).